### PR TITLE
"Uneject" app

### PR DESCRIPTION
**Pull request by uxlayouts**
Un-Ejecting, Adding Custom Linting via CLI tool and SCSS Watch with node-scss-chokidar.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,11 +0,0 @@
-[*]
```
Comment (ceanela) [inline .editorconfig]: Please add this file back into your branch

**Comment by ceanela** [comment]
Hello John, you have several conflicts in your branch, I will be going through, one by one to assist in what needs to be updated.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (ceanela) [inline .eslintrc.json]: We currently already have linting in the project, it was completed in this card: https://trello.com/c/Z85kucc3/7-add-linting-to-project

File: package0.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
+{
```
Comment (ceanela) [inline package0.json]: I'm not sure where this file came from, but let's limit extraneous, ambiguous files. Please make changes in the original `package.json`

File: src/components/Icon.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import React, {Component} from 'react'
+import React from "react"
```
Comment (ceanela) [inline src/components/Icon.js]: You may not have latest, we are committed to the `airbnb` style guide of JS. Single quotes, 2 spaces, etc. We had a discussion yesterday in the dev channel. I apologize if we did not tag you on the convo.

Here's the [Airbnb JS style guide](https://github.com/airbnb/javascript)

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,8 +0,0 @@
-@import 'includes';
```
Comment (ceanela) [inline src/styles/index.scss]: I'm sure @stubelshe would not be happy not to have this file. Please revert this change

**Comment by uxlayouts** [comment]
Removing this PR as we have a better one above.
